docs/clusterobservability.md

```diff
@@ -24,20 +24,29 @@ metadata:
   name: logging-tracing
   namespace: observability
 spec:
-  storage:
-    objectStorage:
-      s3:
-        bucket: bucket-name
-        endpoint: http://minio.minio.svc:9000
-        accessKeyID: tempo
-        accessKeySecret:
-          name: minio-secret
-          key: access_key_secret
   capabilities:
     logging:
       enabled: true
+      storage:
+        objectStorage:
+          s3:
+            bucket: loki 
+            endpoint: http://minio.minio.svc:9000
+            accessKeyID: tempo
+            accessKeySecret:
+              name: minio-secret
+              key: access_key_secret
     tracing:
       enabled: true
+      storage:
+        objectStorage:
+          s3:
+            bucket: tempo
+            endpoint: http://minio.minio.svc:9000
+            accessKeyID: tempo
+            accessKeySecret:
+              name: minio-secret
+              key: access_key_secret
```

Comment:
I would argue to keep the single storage spec. Having to specify per capability just makes things more complicated, especially with each capability supporting a different spec.
But perhaps I'm missing the motivation for this change?